src/elements/Icon/IconGroup.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react'
+import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+import _Icon from './_Icon'
+
+function IconGroup(props) {
+  const {
+    children,
+  } = props
+
+  const rest = getUnhandledProps(IconGroup, props)
+
+  return (
+    <_Icon {...rest} _sdClass='sd-icon-group' _iconClass='icons'>
```

Comment:
I don't think we should reuse the _Icon here.  I tried all the available [icon classes](http://semantic-ui.com/elements/icon.html#/definition) on the `icons` group and only the size classes work.

In order to preserve the propTypes validation, I think we should have a separate IconGroup component that only implements the `size` prop.